_data/versions.yml

```diff
@@ -701,6 +701,35 @@ v3.4:
       url: https://github.com/projectcalico/pod2daemon/releases/tag/v3.4.0
 
 v3.3:
+- title: v3.3.7
+  components:
+    calico/node:
+      version: v3.3.7
+      url: https://github.com/projectcalico/node/releases/tag/v3.3.7
+    calicoctl:
+      version: v3.3.7
+      url: https://github.com/projectcalico/calicoctl/releases/tag/v3.3.7
+    calico/cni:
+      version: v3.3.7
+      url: https://github.com/projectcalico/cni-plugin/releases/tag/v3.3.7
+    calico/kube-controllers:
+      version: v3.3.7
+      url: https://github.com/projectcalico/kube-controllers/releases/tag/v3.3.7
+    networking-calico:
+      version: v3.3.0
+      url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=3.3.0
```

Comment:
hmmm this isn't a viewable webpage. It might work with the git tool.
I couldn't find a 'release' page like github. We could change to https://opendev.org/openstack/networking-calico/src/tag/3.3.0. WDYT?

Reply:
Thanks, that suggestion LGTM - I'll fix this here and in my other PRs (and elsewhere in the file).